x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/models_management/expanded_row.tsx

```diff
@@ -0,0 +1,365 @@
+                <EuiTitle size={'xs'}>
+                  <h5>
+                    <FormattedMessage
+                      id="xpack.ml.inference.modelsList.expandedRow.detailsTitle"
+                      defaultMessage="Details"
+                    />
+                  </h5>
+                </EuiTitle>
+                <EuiSpacer size={'m'} />
+                <EuiDescriptionList
+                  compressed={true}
+                  type="column"
+                  listItems={formatToListItems(details)}
+                />
+              </EuiPanel>
+            </EuiFlexItem>
+          </EuiFlexGrid>
+        </>
+      ),
+    },
+    ...(inferenceConfig
+      ? [
+          {
+            id: 'config',
+            name: (
+              <FormattedMessage
+                id="xpack.ml.inference.modelsList.expandedRow.configTabLabel"
+                defaultMessage="Config"
+              />
+            ),
+            content: (
+              <>
+                <EuiSpacer size={'m'} />
+                <EuiFlexGrid columns={2} gutterSize={'m'}>
+                  <EuiFlexItem>
+                    <EuiPanel>
+                      <EuiTitle size={'xs'}>
+                        <h5>
+                          <FormattedMessage
+                            id="xpack.ml.inference.modelsList.expandedRow.inferenceConfigTitle"
+                            defaultMessage="Inference configuration"
+                          />
+                        </h5>
+                      </EuiTitle>
+                      <EuiSpacer size={'m'} />
+                      <EuiDescriptionList
+                        compressed={true}
+                        type="column"
+                        listItems={formatToListItems(
+                          inferenceConfig[Object.keys(inferenceConfig)[0]]
+                        )}
+                      />
+                    </EuiPanel>
+                  </EuiFlexItem>
+                  {metadata?.analytics_config && (
+                    <EuiFlexItem>
+                      <EuiPanel>
+                        <EuiTitle size={'xs'}>
+                          <h5>
+                            <FormattedMessage
+                              id="xpack.ml.inference.modelsList.expandedRow.analyticsConfigTitle"
+                              defaultMessage="Analytics configuration"
+                            />
+                          </h5>
+                        </EuiTitle>
+                        <EuiSpacer size={'m'} />
+                        <EuiDescriptionList
+                          compressed={true}
+                          type="column"
+                          listItems={formatToListItems(metadata.analytics_config)}
+                        />
+                      </EuiPanel>
+                    </EuiFlexItem>
+                  )}
+                </EuiFlexGrid>
+              </>
+            ),
+          },
+        ]
+      : []),
+    {
+      id: 'stats',
```

Comment:
As I use a reindex pipeline, the stats shown in the expanded row don't seem to update after the first time the row is expanded. I had to do a full browser refresh to show the updated stats:

![image](https://user-images.githubusercontent.com/7405507/90625434-78f70f00-e211-11ea-9014-d1188151785d.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/74978/commits/5985295f9869309c7747c7492421cf7b43becb4a